src/gateway/_types.ts

```diff
@@ -0,0 +1,5 @@
+export const networkIdMap = new Map<number, string>()
```

Comment:
I'm dreaming of `@radixdlt/network-conifg` js repository because this seems to be redefined in almost every project we have  🌚

Reply:
Maybe some day... maybe